hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -862,8 +862,13 @@ public AclStatus getAclStatus(final Path path) throws IOException {
    */
   @Override
   public void access(final Path path, FsAction mode) throws IOException {
```

Comment:
Should {{getIsNamespaceEnabled()}} be checked? In the test, you use modifyAclEntries which requires hierarchical namespace to be enabled. Shouldn't this be the case for access() as well?